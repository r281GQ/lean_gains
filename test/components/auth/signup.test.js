import React from 'react';
import { shallow } from 'enzyme';
import SignUp from './../../../src/components/auth/signup';

describe('calorie_target_result_sum_label', () => {
  it('should render', () => {
    const component = shallow(
      <SignUp
        required={value => () => value}
        signUpHandler={value => value}
        validateEmail={value => value}
        validatePasswordAgain={value => value}
      />
    );
    expect(component).toHaveLength(1);
    expect(component).toMatchSnapshot();
  });
});
