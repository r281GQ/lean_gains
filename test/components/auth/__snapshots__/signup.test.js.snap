// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`calorie_target_result_sum_label should render 1`] = `
<div
  className="sign-up__container"
>
  <Paper
    circle={false}
    className="sign-up__paper"
    rounded={true}
    transitionEnabled={true}
    zDepth={1}
  >
    <FlatButton
      disabled={true}
      fullWidth={false}
      label="Sign up"
      labelPosition="after"
      labelStyle={Object {}}
      onKeyboardFocus={[Function]}
      onMouseEnter={[Function]}
      onMouseLeave={[Function]}
      onTouchStart={[Function]}
      primary={false}
      secondary={false}
    />
    <form
      onSubmit={[Function]}
    >
      <div>
        <Field
          component={[Function]}
          floatingLabelText="Username"
          name="userName"
          validate={[Function]}
        />
      </div>
      <div>
        <Field
          component={[Function]}
          floatingLabelText="Email address"
          name="email"
          type="email"
          validate={
            Array [
              [Function],
              [Function],
            ]
          }
        />
      </div>
      <div>
        <Field
          component={[Function]}
          floatingLabelText="Password"
          name="password"
          type="password"
          validate={[Function]}
        />
      </div>
      <div>
        <Field
          component={[Function]}
          floatingLabelText="Password once more"
          name="passwordAgain"
          type="password"
          validate={[Function]}
        />
      </div>
      <div
        className="sign-up-button__container "
      >
        <RaisedButton
          disabled={false}
          fullWidth={true}
          label="Sign Up"
          labelPosition="after"
          primary={true}
          secondary={false}
          type="submit"
        />
      </div>
    </form>
  </Paper>
</div>
`;
